scss/utilities/_utilities.scss

```diff
@@ -0,0 +1,79 @@
+// Property utilities generation
+
+
+// Loop over every breakpoint
+@each $breakpoint in map-keys($grid-breakpoints) {
+  // Generate media query if needed
+  @include media-breakpoint-up($breakpoint) {
+    $infix: breakpoint-infix($breakpoint, $grid-breakpoints);
+    // Loop over each utility property
+    @each $key, $utility in $utilities {
+
+      // Only render if responsive media queries are enabled or if it's the base media query
+      @if type_of($utility) == "map" and (map_get($utility, responsive) or $infix == "") {
+        $values: map_get($utility, values);
+
+        // If the values are a list or string, convert it into a map
```

Comment:
Could we remove this check if we only accepted maps for the values?

Reply:
I did the type check because of:
https://github.com/twbs/bootstrap/blob/b1a7b1a335e813bc030b4fa4918f9b826445f9e2/scss/_utilities.scss#L50-L53

This was the simplest way to disable it

Reply:
I clarified this now: https://github.com/twbs/bootstrap/blob/a41ab9db631e187bc01b4654bbf620cd60eeb286/scss/utilities/_utilities.scss#L45